### ISSUE TITLE
Clerics and televangelists also for other governments

### ISSUE THREAD

**Issue by LynxAbraxas** [civctp2/civctp2#129]
Issue for discussing what was started in https://github.com/civctp2/civctp2/issues/112#issuecomment-488147133.

**Comment by LynxAbraxas** [comment]
One option to allow clerics and televangelists to other govs while keeping theocracy as an interesting gov would be to make clerics and televangelists cheeper to built in theocracy or limiting their numbers or their success in other govs. @MartinGuehmann how much effort would it be to implement such?
A pro for clerics and televangelists to be available to other govs would be that it would surely make the game more interesting.

**Comment by MartinGuehmann** [comment]
> One option to allow clerics and televangelists to other govs while keeping theocracy as an interesting gov would be to make clerics and televangelists cheeper to built in theocracy or limiting their numbers or their success in other govs. @MartinGuehmann how much effort would it be to implement such?

I don't think it is too much, E put in this Government modified stuff. The only thing is to implement it properly. It requires quite some repetitive code changes. The alternative is to add a government specific discount.

> A pro for clerics and televangelists to be available to other govs would be that it would surely make the game more interesting.

Sure it would make the game more interesting. And of course to make Theocracy more interesting Fascism should indeed come later. So we can already agree on this.

**Comment by LynxAbraxas** [comment]
> The alternative is to add a government specific discount.

How about adding a gov discount and an increased chance of success of their actions?

> Fascism should indeed come later. So we can already agree on this.

I would welcome such a change.

**Comment by LynxAbraxas** [comment]
Possibly part of #186: https://github.com/civctp2/civctp2/pull/186/files#diff-7e6597dac40622ec01209766e48d9c92

**Comment by LynxAbraxas** [comment]
> Sure it would make the game more interesting. And of course to make Theocracy more interesting Fascism should indeed come later. So we can already agree on this.

Hm, could that be realized by just re-ordering here:
https://github.com/civctp2/civctp2/blob/94e4ff7070cf1503909558151467dc993957bb31/ctp2_data/default/aidata/strategies.txt#L126-L129

**Comment by MartinGuehmann** [comment]
> > Sure it would make the game more interesting. And of course to make Theocracy more interesting Fascism should indeed come later. So we can already agree on this.
> 
> Hm, could that be realized by just re-ordering here:
> https://github.com/civctp2/civctp2/blob/94e4ff7070cf1503909558151467dc993957bb31/ctp2_data/default/aidata/strategies.txt#L126-L129

No that is just about what government the AI selects. The strategies don't care about what the AI can choose from. That is determined by the governments available. The AI selects the first government available on that list in this Monarchy. This is from the default strategy other AIs may have Theocracy first. So those will switch to Theocracy, but not those that use the default strategy.

Anyway, reordering here won't help the AI with getting Theocracy, since it comes late and Fascism is just a few turns away.

**Comment by MartinGuehmann** [comment]
I merged your PR. Thus this can be closed. Unfortunately, I couldn't link this to the PR, so that I have to close it manually.

**Comment by LynxAbraxas** [comment]
Hm, my #410 solved only the bug that clerics and televangelists were not built by the AI if the player is in theocracy.
As I understand, this issue is for discussing how other goverments than theocracy could be given access to build clerics and televangelists (possibly at a higher price and/or lower success rate). Therefore I'd suggest to open this issue up again (seems I can't do that).

**Comment by MartinGuehmann** [comment]
Well then. I meant my original PR to allow the AI to build clerics. One problem was that the AI was not long enough in theology. The problem with the clerics, I consider be solved.

The televangelists, just need to be moved into a later government. Maybe virtual democracy. But that is now just a guess.
